docs/source/services/services.rst

```diff
@@ -0,0 +1,256 @@
+        | secret_key     | 32 byte secret key to use to generate and validate JWTs           |  **Required**   |
+        +----------------+-------------------------------------------------------------------+-----------------+
+        | algorithm      | Algorithm to use generate and validate JWTs (default: HS256)      |  **Optional**   |
+        +----------------+-------------------------------------------------------------------+-----------------+
+        | users          | List of user entries allowed to use the REST service. (see below) | **Required**    |
+        +----------------+-------------------------------------------------------------------+-----------------+
+
+    .. table:: Options available to the user entries
+
+        +-----------------+-------------------------------------------------+-----------------+
+        | Option          | Short Description                               | Requirement     |
+        +=================+=================================================+=================+
+        | user_name       | The name of the user                            | **Required**    |
+        +-----------------+-------------------------------------------------+-----------------+
+        | hashed_password | The ``bcrypt`` hashed password of that user     | **Required**    |
+        +-----------------+-------------------------------------------------+-----------------+
+        | scopes          | List of scopes the user is allowed to request.  | **Required**    |
+        +-----------------+-------------------------------------------------+-----------------+
+
+    .. note::
+
+        The REST service is using a 32 byte ``secret_key`` to generate and verify JWTs. It can be created using
+        ``openssl rand -hex 32``.
+
+.. content-tabs:: right-col
+
+    .. rubric:: Example configuration
+
+    .. code-block:: yaml
+
+        Services:
+          restapi:
+            !TardisRestApi
+            host: 127.0.0.1
+            port: 1234
+            secret_key: 752e003f636f402cc23728e185ce8c9eef27b7e02cf509b3015f7757e625b8e4
+            algorithm: HS256
+            users:
+              - user_name: tardis
+                hashed_password: $2b$12$c9SSllh1U6tOhIo37sDWF.kdRIU5RQAAOHL9bVYMs2.HluyFE43Uq
+                scopes:
+                 - resources:get
+        Plugins:
+          SqliteRegistry:
+            db_file: drone_registry.db
+
+Available logging configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. content-tabs:: left-col
+
+    The REST service uses ``uvicorn`` as ASGI server, which provides some additional logging functionality. By default
+    the access and errors logs are written into the regular ``TARDIS`` logfile. However, it can be configured to write
+    both access and error logs into a separate file. Therefore, an ``uvicorn`` logger needs to be configured. An example
+    can be found below.
```

Comment:
Can we actually ensure that the example is rendered below? Maybe it is better to state something like "See the example logging configuration.".

Reply:
Thanks, I thing this is a better.